### PR TITLE
slight reword of immediate spending

### PR DESCRIPTION
**Pull request by sternhenri**
follow-up to https://github.com/filecoin-project/specs/pull/368

### CODE REVIEW

File: mining.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ As mentioned above, every round, a miner checks to see if they have been selecte
 
 Other nodes will receive these blocks and form a TipSet out of the eligible blocks (those that have the same parents and are at the same block height). These nodes will then validate the TipSet. The full procedure for how to verify a TipSet can be found above in [Block Validation](#block-validation). To validate TipSet state, the validating node will, for each block in the TipSet, first apply the block reward value directly to the mining node's account and then apply the messages contained in the block.
 
-Thus, each of the miners who produced a block in the TipSet will receive a block reward. There will be no lockup. These rewards can be spent immediately. 
+Thus, each of the miners who produced a block in the TipSet will receive a block reward. There will be no lockup imposed as a network-wide parameter, though it is assumed some clients will choose to set their own freezing period based on their own security needs (and Filecoin's confirmation time). In cases this does not happen, these rewards can be spent immediately.
```
Comment (dignifiedquire) [inline mining.md]: - who are "clients" in this context?
- what is "freezing period"? 
- how can such a freeze be set?

Comment (sternhenri) [reply mining.md]: Attempted to resolve these in new commit. Lmk what you think.

For the third question (how can such a freeze be set): the answer is likely through custom node rules, just as BTC conf time rules are set in modern wallets. But I'd rather not go into it in the spec.... what do you think?

**Comment by sternhenri** [comment]
That's a fair concern. Feel free to close unmerged!

**Comment by sternhenri** [comment]
Close bc confusing.

**Pull request closed without merging**